packages/ui/makeup-combobox/dist/cjs/index.js

```diff
@@ -169,9 +172,18 @@ function _onListboxClick(e) {
   const widget = this;
   const element = e.target.closest("[role=option]");
   const indexData = this._listboxWidget.items.indexOf(element);
-  console.log(indexData);
   if (indexData !== undefined) {
     this._inputEl.value = this._listboxWidget.items[indexData].innerText;
+
+    // TODO: refactor this redundant logic
+    if (this._autocompleteType === "list") {
```

Comment:
Had to manually filter these items again for click as the textBoxInput event wont be triggered with setting the value. Another option is to manually dispatch the input event